roles/contiv_network/tasks/main.yml

```diff
@@ -82,8 +85,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+- name: reload system config for netmaster
+  shell: systemctl daemon-reload
+  when: run_as == "master"
```

Comment:
Given that etcd doesn't have the reload and still works fine, I wonder if it is required at all. I don't see any harm in adding it though, so will go ahead with it.